http/oidc/src/main/java/org/wildfly/security/http/oidc/Oidc.java

```diff
@@ -52,10 +52,14 @@ public class Oidc {
     public static final String TEXT_CONTENT_TYPE = "text/*";
     public static final String DISCOVERY_PATH = ".well-known/openid-configuration";
     public static final String KEYCLOAK_REALMS_PATH = "realms/";
+    public static final String KEYSTORE_PASS = "password";
```

Comment:
Is this something that's used for tests? It shouldn't be added to the constants class.

Reply:
moved relevant constants to KeycloakConfiguration class.